public/static-checkout/js/scripts.js

```diff
@@ -250,27 +239,39 @@ const countryList = {
   TN: 'Tunisia',
   TR: 'Turkey',
   TM: 'Turkmenistan',
-  TC: 'Turks and Caicos Islands (the)',
+  TC: 'Turks and Caicos Islands',
   TV: 'Tuvalu',
   UG: 'Uganda',
   UA: 'Ukraine',
-  AE: 'United Arab Emirates (the)',
-  GB: 'United Kingdom of Great Britain and Northern Ireland (the)',
-  UM: 'United States Minor Outlying Islands (the)',
-  US: 'United States of America (the)',
+  AE: 'United Arab Emirates',
+  GB: 'United Kingdom',
+  US: 'United States of America',
+  UM: 'United States Minor Outlying Islands',
   UY: 'Uruguay',
   UZ: 'Uzbekistan',
   VU: 'Vanuatu',
-  VE: 'Venezuela (Bolivarian Republic of)',
+  VE: 'Venezuela',
   VN: 'Viet Nam',
-  VG: 'Virgin Islands (British)',
-  VI: 'Virgin Islands (U.S.)',
+  VG: 'Virgin Islands, British',
+  VI: 'Virgin Islands, U.S.',
   WF: 'Wallis and Futuna',
   EH: 'Western Sahara',
   YE: 'Yemen',
   ZM: 'Zambia',
   ZW: 'Zimbabwe',
   AX: 'Åland Islands',
+  BQ: 'Bonaire, Sint Eustatius and Saba',
```

Comment:
I'm surprised the list I got was so out of date. Let's do this in alphabetical order?